jupyter_core/command.py

```diff
@@ -70,6 +83,37 @@ def main():
         parser = jupyter_parser()
         args, opts = parser.parse_known_args()
         subcommand = args.subcommand
+        if args.json and not args.paths:
+            print("--json is only used with --paths", file=sys.stderr)
+            sys.exit(1)
```

Comment:
There's a handy shortcut for this - `sys.exit('string')` will print the string to stderr and exit with code 1.